src/game-loop/src/populator/Populator.cpp

```diff
@@ -203,6 +210,16 @@ void populator::generate_loot(std::shared_ptr<LevelSummaryTracker>& tracker)
             switch (loot_type)
             {
                 case LootType::NOTHING: break;
+                case LootType::SHOP_ITEM:
+                {
+                    // FIXME: Just a placeholder for now:
+                    auto entity = prefabs::Shotgun::create(pos_x, pos_y);
+                    // Add SaleableComponent to it:
+                    //SaleableComponent saleable(0, entt::null, entity);
+                    //saleable.add_animation();
+                    registry.emplace<SaleableComponent>(entity, 1000, entt::null, entity);
```

Comment:
Think of some helper to generate shop contents and assigning price for each item

Reply:
Requirements for the helper:

* Takes item count
* Returns a list of entities (items with already attached `SaleableComponent`)
* Must already assign price to each of them
* Must accept a theme for the returned items, i.e `ShoppingTheme::WEAPONS` or `ShoppingTheme::CLOTHES`